### PR TITLE
added AnirudhPanda

### LINKED ISSUE #596: Anirudh Panda
pro

### PR DESCRIPTION
**Pull request by AnirudhPanda**
resolve #596

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thank you! 😄:+1:
@schmelto, this PR can be merged.

File: data.json [hunk 1]
```diff
@@ -1118,13 +1118,19 @@
   {
     "name": "Victoria Lo",
     "githubUsername": "victoria-lo",
-    "imageUrl": "https://user-images.githubusercontent.com/47699978/125807703-6d8c6adf-d642-42f2-b1bf-51f6a32c0a84.png", 
+    "imageUrl": "https://user-images.githubusercontent.com/47699978/125807703-6d8c6adf-d642-42f2-b1bf-51f6a32c0a84.png",
```
Comment (schmelto) [inline data.json:1121]: What is changed here?

Comment (AnirudhPanda) [reply data.json:1121]: Nothing, I think extra spaces

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for collaborating and contributing

**Comment by AnirudhPanda** [comment]
Hey @schmelto can you review this PR, already getting too much merge conflicts

**Comment by kaiwalyakoparkar** [comment]
Sorry for the inconvenience @AnirudhPanda. Thanks for contributing :)

File: data.json [hunk 2]
```diff
@@ -1133,16 +1133,30 @@
...
     "issueNumber": 600
-  },
+
+
+  
+
+  
+
+
+
```
Comment (schmelto) [inline data.json:1154]:
```suggestion
```

File: data.json [hunk 3]
```diff
@@ -1133,16 +1133,30 @@
...
+  
   {
     "name": "Rohit Yadav",
     "githubUsername": "rohitturbot",
     "imageUrl": "https://user-images.githubusercontent.com/40739903/126065686-40212b44-01a8-4c5f-b38a-de22ca76cc9c.png",
     "issueNumber": 602
   }
 ]
+
```
Comment (schmelto) [inline data.json:1162]:
```suggestion
```

**Review by schmelto — COMMENTED** [review]
Why it was merged with empty lines?

**Pull request merged**